src/Feedback.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Gorse;
+
+class Feedback implements \JsonSerializable
+{
+    public function __construct(
+        public string $feedback_type,
```

Comment:
constructor property promotion has been introduced in php 8 but composer allow you to use 7.4

Reply:
Sorry, I overlooked this point. Thank you for the reminder. I will handle it as soon as possible.